lib/class-wp-rest-menu-items-controller.php

```diff
@@ -368,64 +478,51 @@ protected function prepare_item_for_database( $request ) {
 				$check = rest_validate_value_from_schema( $request[ $api_request ], $schema['properties'][ $api_request ] );
```

Comment:
Let's remove this double schema validation while we're here. The request data has already been validated and sanitized according to the schema.